CHANGELOG.md

```diff
@@ -79,13 +79,15 @@ The table below shows which release corresponds to each branch, and what date th
 - [#2444][2444] Add `ELF.close()` to release resources
 - [#2413][2413] libcdb: improve the search speed of `search_by_symbol_offsets` in local libc-database
 - [#2470][2470] Fix waiting for gdb under WSL2
+- [#2478][2478] libcdb-cli: changes for previous updates
```

Comment:
Can you describe the changes in a way that people reading the changelog understand what was changed without looking at the code please?